src/components/ItemsNotFoundComponent.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { Image } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import '../_css/ItemsNotFoundComponent.css';
+
+export class ItemsNotFoundComponent extends Component {
+  state = {
```

Comment:
This `state` is not needed. Use the message from props. Read more about derived state [here](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html)